index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap"
+      rel="stylesheet"
+    />
+    <style>
+      body {
+        width: 375px;
+        height: 667px;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Quiz App</h1>
+    </header>
+    <main>
+      <div class="questionBox">
+        <img
+          class="img-bookmark-white"
+          src="bookmark-white.svg"
+          alt="bookmark"
+        />
+        <section class="questionCard">
+          <p>What property flips the axes in flexbox?</p>
+          <button class="button">Show Answer</button>
+          <div class="tags">
+            <a class="tag1" href="index.html">#html</a>
+            <a class="tag2" href="flexbox.html">#flexbox</a>
+            <a class="tag3" href="styles.css">#css</a>
```

Comment:
you don't need to add numbers to the classes. They are meant to be reused! 
```suggestion
            <a class="tag" href="index.html">#html</a>
            <a class="tag" href="flexbox.html">#flexbox</a>
            <a class="tag" href="styles.css">#css</a>
```